### PR TITLE
Add API to return system information

### PR DESCRIPTION
**Pull request by iliajie**

### CODE REVIEW

File: virtual_feature.pl [hunk 1]
```diff
@@ -3507,4 +3507,31 @@ sub feature_delete_protected_dir
 return wantarray ? ($err, $status) : $err;
 }
 
+# feature_sysinfo()
+# Returns Nginx and available PHP version
+sub feature_sysinfo
+{
+my @rv = ( [ $text{'sysinfo_nginx'}, &get_nginx_version() ] );
+if (defined(&virtual_server::list_available_php_versions)) {
```
Comment (jcameron) [inline virtual_feature.pl:3515]: When would this not be defined?

Comment (iliajie) [reply virtual_feature.pl]: Never. It's always defined. Fixed now.

File: virtual_feature.pl [hunk 2]
```diff
@@ -3507,4 +3507,29 @@ sub feature_delete_protected_dir
...
+# feature_sysinfo()
+# Returns Nginx and available PHP version
+sub feature_sysinfo
+{
+my @rv = ( [ $text{'sysinfo_nginx'}, &get_nginx_version() ] );
+my @avail = &virtual_server::list_available_php_versions();
+my @vers;
+foreach my $a (grep { $_->[1] } @avail) {
+	&clean_environment();
+	my $out = &backquote_command("$a->[1] -v 2>&1 </dev/null");
```
Comment (jcameron) [inline virtual_feature.pl:3519]: There's a method `get_php_version` which will get the version from a PHP executable

Comment (iliajie) [reply virtual_feature.pl]: Thanks! Fixed now!

**Pull request merged**